caiman/mmapping.py

```diff
@@ -73,7 +74,22 @@ def load_memmap(filename: str, mode: str = 'r') -> Tuple[Any, Tuple, int]:
             return (Yr, (d1, d2, d3), T)
     else:
         logging.error("Unknown extension for file " + str(filename))
-        raise Exception('Unknown file extension (should be .mmap)')
+        raise ValueError('Unknown file extension (should be .mmap)')
+
+    # Strip path components and use CAIMAN_DATA/example_movies
```

Comment:
why is this code repeated here?

Reply:
That's an artifact from the `merge` operation I did, sorry about that.